_universum/modules/artifact_collector.py

```diff
@@ -99,28 +99,27 @@ def __init__(self, *args, **kwargs):
     def make_file_name(self, name):
         return utils.calculate_file_absolute_path(self.artifact_dir, name)
 
-    def create_text_file(self, name):
+    def create_text_file(self, name: str):
         try:
             file_name = self.make_file_name(name)
 
-            # File system interaction modules only work with encoded unicode strings
             if isinstance(file_name, six.text_type):
-                encoded_name = file_name.encode("utf-8")
-            else:
-                encoded_name = str(file_name)
+                file_name = str(file_name)
+            assert str == type(file_name)
```

Comment:
isinstance?

Reply:
no, I want exact checking here

Reply:
Please discuss using asserts and don't resolve the conversation